.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: 'FEATURE REQUEST: '
+labels: ':type: enhancement'
+assignees: ''
+
+---
+
+*NOTE: This issue template is being created and may not be complete.*
+
+*NOTE: Please only submit feature requests to this repository for features
+requested for the testing repository itself. General feature requests for Rocky
+Linux should be submitted for general discussion in the [Rocky Linux
+Mattermost](https://chat.rockylinux.org) or [Rocky Linux
+Forums](https://forums.rockylinux.org).*
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always
+frustrated when [...]
```

Comment:
I'm not sure if I would rather keep negative statements out or include it as a list of several examples:
* It would improve my workflow if...
* It impedes my work when...
Thoughts?

Reply:
Awesome... you can use the 'Insert a suggestion' widget to commit directly to the code and get attribution (like you care).